src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -80,7 +80,7 @@ public function getListeners($eventName = null)
      */
     public function hasListeners($eventName = null)
     {
-        return (bool) count($this->getListeners($eventName));
+        return (bool) $this->getListeners($eventName);
```

Comment:
In that case you would have same problem with count
https://3v4l.org/goLns

Point of this PR is that EventDispatcher counts number of listeners needlessly. It only needs to know if it's not empty, it doesn't need to count all of them first.